CMakeLists.txt

```diff
@@ -4,62 +4,34 @@ project(roboteam_world)
 # Needed for the documentation generator.
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
-## Find catkin macros and libraries
-## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
-## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  rospy
-  roslib
-  std_msgs
-  roboteam_msgs
-  roboteam_utils
-)
 find_package(Armadillo REQUIRED)
 include_directories(${ARMADILLO_INCLUDE_DIRS})
 
-## Enable c++11 support.
-set(CMAKE_CXX_FLAGS "-std=c++11 -Wall -g ${CMAKE_CXX_FLAGS} -O0")
+## Enable c++17 support.
+set(CMAKE_CXX_FLAGS "-std=c++17 -Wall -g ${CMAKE_CXX_FLAGS} -O0")
```

Comment:
it was already there but use of ${CMake_CXX_FLAGS} is really ugly. If the flags have O1 for release set for example, will it get overwritten now or not?